### PR TITLE
Added device_id to Kakfa message payload.

### PR DESCRIPTION
**Pull request by jasebell**

### CODE REVIEW

File: src/kixi/hecuba/weather/service.clj [hunk 1]
```diff
@@ -31,6 +32,9 @@
                                        (.getBytes (gen-message
                                                    (get entity "entity_id")
                                                    (get entity "property_code")
+                                                   (-> (get-in entity ["devices"])
+                                                       first
+                                                       (get "device_id"))
```
Comment (otfrom) [inline src/kixi/hecuba/weather/service.clj:39]: are you sure it is always going to be the first device you want?

Comment (jasebell) [inline src/kixi/hecuba/weather/service.clj:39]: Yes.

File: src/kixi/hecuba/weather/service.clj [hunk 2]
```diff
@@ -18,28 +18,37 @@
...
   (let [url-to-get (str api-endpoint
                         "entities/?q=property_type:\""
                         entity-type
-                        "\"&page=0&size="
+                        "\" AND programme_id:"
+                        programme-id
+                        "&page=0&size="
                         max-entries-per-page
                         "&sort_key=programme_name.lower_case_sort&sort_order=asc")]
+    (println url-to-get)
```
Comment (elisehuard) [inline src/kixi/hecuba/weather/service.clj]: debugging msg?

Comment (jasebell) [inline src/kixi/hecuba/weather/service.clj]: Good catch. Thanks.

**Comment by elisehuard** [comment]
lgtm :shipit:

**Pull request merged**